test/Project/project.csproj

```diff
@@ -2,12 +2,12 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
.NET Core 2.2 reached EoL on December 23, 2019.
Upgrade to .NET 6.0?

Reply:
@SIkebe  Sorry too late!
Today,  I finally upgraded it to .NET 6.0.

https://github.com/jsakamoto/nupkg-selenium-webdriver-iedriver/commit/2baab4caf1a05f2bc9ec66a7e1391f4feb6a4602?diff=split#diff-e38191792819e929c1fa8780a70de32bbf1fc43b35756f9b442bc6d59530a67aR5